.github/workflows/deploy-dev.yml

```
@@ -107,8 +113,8 @@ jobs:
       - name: Update MYSQL_URL in Parameter Store
         run: |
           MYSQL_URL="jdbc:mysql://${PRIVATE_IP}:3306/eatda?useUnicode=true&characterEncoding=UTF-8"
-
-          aws ssm put-parameter \
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)